infra-templates/gandi-dns/0/rancher-compose.yml

```diff
@@ -0,0 +1,41 @@
+.catalog:
+  name: "Gandi DNS"
+  version: "v0.1.0"
+  description: "Rancher External DNS service powered by Gandi. Requires Rancher version v1.1.0"
+  minimum_rancher_version: v1.1.0
+  #maximum_rancher_version: 
+  questions:
+    - variable: "GANDI_APIKEY"
+      label: "Gandi API key"
+      description: "API key for your Gandi account"
+      type: "string"
+      required: true
+    - variable: "ROOT_DOMAIN"
+      label: "Root Domain"
+      description: "The domain name managed by Gandi."
+      type: "string"
+      required: true
+    - variable: "TTL"
+      label: "TTL"
+      description: "The resource record cache time to live (TTL), in seconds (minimum 300)"
+      type: "int"
+      default: 1500
```

Comment:
Any reason you are not using the minimum TTL as default? 5 mins is already quite high for dynamic environments.